test/vc-data-model-1.0/10-basic.js

```diff
@@ -218,22 +218,22 @@ describe('Basic Documents', function() {
       expect(hasType(doc, 'VerifiablePresentation')).to.be.true;
     });
 
-    it('MUST include `verifiableCredential` and `proof`', async function() {
+    it('MAY include `verifiableCredential` and `proof`', async function() {
       const doc = await util.generatePresentation('example-8.jsonld', generatorOptions);
       should.exist(doc.verifiableCredential);
       should.exist(doc.proof);
     });
 
-    it('MUST include `verifiableCredential` and `proof` (negative - missing `verifiableCredential`)', async function() {
+    it('`proof`, if present, MUST include `type` property (negative)', async function() {
```

Comment:
I see thee test here... I know the intent, just feels like the cases might be described in clearer langauge.

Reply:
Like this?
```
```